Makefile

```diff
@@ -162,14 +149,68 @@ full-reset: docker-compose-rm clean all
 
 .PHONY: docker
 docker:
-	docker build -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) .
+	docker buildx build \
+		--load \
+		-t "$(DOCKER_TAG)" \
+		-t "$(DOCKER_TAG):parent" \
+		--target=$(DOCKER_TARGET) \
+		--build-arg=GIT_VERSION=$(GIT_VERSION) \
+		--build-arg=BUILD_TARGET=$(BUILD_TARGET) \
+		--build-arg=FROM_LOCAL_PARENT=$(FROM_LOCAL_PARENT) \
+		$(ARGS) \
+		.
 
 .PHONY: docker-local
-docker-local:
-	tar ch ./bin Dockerfile.local | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
-
+docker-local: downloader livepeer-log scripts 
+	tar ch ./bin Dockerfile.local ./config \
+	| docker buildx build \
+		--load \
+		-f Dockerfile.local \
+		-t "$(DOCKER_TAG)" \
+		--build-arg=GIT_VERSION=$(GIT_VERSION) \
+		--build-arg=BUILD_TARGET=$(BUILD_TARGET) \
+		--build-arg=FROM_LOCAL_PARENT=$(FROM_LOCAL_PARENT) \
+		$(ARGS) \
+		-
+
+.PHONY: box
+box: DOCKER_TAG=livepeer/in-a-box
+box: DOCKER_TARGET=livepeer-in-a-box
+box: scripts docker
+
+.PHONY: box-local
+box-local: DOCKER_TAG=livepeer/in-a-box
+box-local: DOCKER_TARGET=livepeer-in-a-box
+box-local: FROM_LOCAL_PARENT=livepeer/in-a-box:parent
+box-local: docker-local
+
+.PHONY: test
 test: docker
 	go test ./test/e2e/*.go -v --logtostderr
 
+.PHONY: test-local
 test-local: docker-local
 	go test ./test/e2e/*.go -v --logtostderr
+
+.PHONY: scripts
+scripts:
+	cp -Rv ./scripts/* ./bin
+
+.PHONY: box-dev
+box-dev: scripts
+	ulimit -c unlimited \
+	&& exec docker run \
+	-v $$(realpath bin):/usr/local/bin \
+	-v $$(realpath data):/data \
+	-v $$(realpath config):/etc/livepeer:ro \
+	-v $$(realpath ./coredumps):$$(realpath ./coredumps) \
+	-e CORE_DUMP_DIR=$$(realpath ./coredumps) \
+	--rm \
+	-it \
+	--name catalyst \
+	--shm-size=4gb \
+	-p 8888:8888 \
+	-p 5432:5432 \
+	-p 1935:1935 \
+	-p 4242:4242 \
+	livepeer/in-a-box
```

Comment:
```
$ make box-dev

cp -Rv ./scripts/* ./bin
./scripts/livepeer-cockroach -> ./bin/livepeer-cockroach
./scripts/livepeer-core-dump-monitor -> ./bin/livepeer-core-dump-monitor
./scripts/livepeer-minio -> ./bin/livepeer-minio
./scripts/livepeer-nginx -> ./bin/livepeer-nginx
./scripts/livepeer-rabbitmq -> ./bin/livepeer-rabbitmq
./scripts/local-livepeer-api -> ./bin/local-livepeer-api
ulimit -c unlimited \
	&& exec docker run \
	-v $(realpath bin):/usr/local/bin \
	-v $(realpath data):/data \
	-v $(realpath config):/etc/livepeer:ro \
	-v $(realpath ./coredumps):$(realpath ./coredumps) \
	-e CORE_DUMP_DIR=$(realpath ./coredumps) \
	--rm \
	-it \
	--name catalyst \
	--shm-size=4gb \
	-p 8888:8888 \
	-p 5432:5432 \
	-p 1935:1935 \
	-p 4242:4242 \
	livepeer/in-a-box
exec /usr/local/bin/catalyst: exec format error
make: *** [box-dev] Error 1
```

Reply:
You're adding the contents of `scripts/` already in Dockerfile `ADD ./scripts /usr/local/bin`.

Reply:
Yeah, but that directory is getting bind-mounted on top of by 
```
-v $(realpath bin):/usr/local/bin \
```
So we need to make sure they get copied over too.